### ISSUE TITLE
Implement central console

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#2033]
When multiple ntopng/nEdge instances are deployed, it is necessary to have a central console that gives the ability to see what the individual ntopng instances are doing including
- current state
- configuration / maintenance ...
- usage stats
- remote reboot / update

This issue depends on #1864 that has to be implemented beforehand in order to allow NAT traversal

**Comment by simonemainardi** [comment]
duplicate of #1991
